src/nxdoc/nuxeo-server/administration/maintenance/hotfixes-installation-notes-for-nuxeo-platform-lts-2021.md

```diff
@@ -103,6 +103,33 @@ If you have any questions, feel free to contact our support team via a dedicated
 
 
 
+## Hotfix 31
+
+### Reduce Retries on Recompute Bulk Actions
+
+The maximum number of retries can now be configured for recompute bulk actions using the nuxeo.conf options below. The new default option is one retry (previously 3):
+```Java
+nuxeo.bulk.action.recomputeThumbnails.maxRetries=1
+nuxeo.bulk.action.recomputeViews.maxRetries=1
+nuxeo.bulk.action.recomputeVideoConversions.maxRetries=1
+```
+
+<i class=fa fa-long-arrow-right aria-hidden=true></i>&nbsp;More on JIRA ticket [NXP-31550](https://jira.nuxeo.com/browse/NXP-31550)
+
+### Add Mongodb Index on ecm:Isversion
+
+For MongoDB backend, create the index manually:
+```Java
+db.default.createIndex(
+   { ecm:isVersion: 1 }
+);
+```
+Otherwise, the nuxeo server will attempt to create this index if not present at start-up.  In the case of an existing instance with large amounts of documents, this process may time out and/or affect performance.
```

Comment:
```suggestion
Otherwise, the nuxeo server will attempt to create this index if not present at start-up. In the case of an existing instance with large amounts of documents, this process may time out and/or affect performance.
```